static/app/utils/replays/hooks/useFetchDatasetTags.tsx

```diff
@@ -0,0 +1,60 @@
+import {useMemo} from 'react';
+
+import {type Dataset, useFetchOrganizationTags} from 'sentry/actionCreators/tags';
+import type {Organization, TagCollection} from 'sentry/types';
+import {FieldKind} from 'sentry/utils/fields';
+
+type UseFetchDatasetTagsParams = {
+  dataset: Dataset;
+  org: Organization;
+  projectIds: string[];
+  enabled?: boolean;
+  end?: string;
+  keepPreviousData?: boolean;
+  start?: string;
+  statsPeriod?: string | null;
+  useCache?: boolean;
+};
+
+/**
+ * Fetches tags from exclusively one dataset. Based on useFetchIssueTags.
+ */
+export default function useFetchDatasetTags({
```

Comment:
Is it worth having this function instead of just using `useFetchOrganizationTags`? Seems like the params are pretty much the same

Reply:
It's mainly to avoid duplicating the useMemo/reduce pattern